docs/verify/simulate-with-qiskit-primitives.mdx

```diff
@@ -5,260 +5,222 @@ description: How to perform exact simulation of quantum circuits using primitive
+from qiskit.primitives import StatevectorEstimator
+estimator = StatevectorEstimator()
 ```
+  </TabItem>
 
-```
-     ┌───────┐
-q_0: ┤ Ry(θ) ├──■──
-     └───────┘┌─┴─┐
-q_1: ─────────┤ X ├
-              └───┘
-```
-
-The main difference with the previous case is that now you need to specify the sets of parameter values for which you want to evaluate the expectation value as a `list` of `list`s of `float`s.
-The `i`<sup>th</sup> element of the outer `list` is the set of parameter values that corresponds to the `i`<sup>th</sup> circuit and observable.
+  <TabItem value="EstimatorV1" label="Estimator (V1)">
+    Instantiate an [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator).
 
 ```python
-import numpy as np
-
-parameter_values = [[0], [np.pi/6], [np.pi/2]]
-
-job = estimator.run([param_qc]*3, [observable]*3, parameter_values=parameter_values)
-values = job.result().values
-
-for i in range(3):
-    print(f"Parameter: {parameter_values[i][0]:.5f}\t Expectation value: {values[i]}")
-```
-
-```
-Parameter: 0.00000   Expectation value: 2.0
-Parameter: 0.52360   Expectation value: 3.0
-Parameter: 1.57080   Expectation value: 4.0
+from qiskit.primitives import Estimator
+estimator = Estimator()
 ```
+  </TabItem>
+</Tabs>
 
-### Change run options
-
-Your workflow might require tuning primitive run options, such as the number of shots.
 
-By default, the reference [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) class performs an exact statevector calculation based on the [`qiskit.quantum_info.Statevector`](../api/qiskit/qiskit.quantum_info.Statevector) class. However, this can be modified to include shot noise if the number of `shots` is set. For reproducibility purposes, a `seed` will also be set in the following examples.
-
-There are two main ways of setting options in the [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator):
+### Run and get results
 
-- Set keyword arguments in the [`qiskit.primitives.Estimator.run`](../api/qiskit/qiskit.primitives.Estimator#run) method.
-- Modify [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator) options.
+This example only uses one circuit (of type [`QuantumCircuit`](../api/qiskit/qiskit.circuit.QuantumCircuit)) and one
+observable.
 
-#### Set keyword arguments for [`qiskit.primitives.Estimator.run`](../api/qiskit/qiskit.primitives.Estimator#run)
 
-If you only want to change the settings for a specific run, it can be more convenient to set the options inside the [`qiskit.primitives.Estimator.run`](../api/qiskit/qiskit.primitives.Estimator#run) method. You can do this by passing them as keyword arguments.
+<Tabs>
+  <TabItem value="EstimatorV2" label="StatevectorEstimator (V2)">
+    Run the estimation by calling the [`StatevectorEstimator.run`](../api/qiskit/qiskit.primitives.StatevectorEstimator#run)
+    method, which returns an instance of ... You can get the results from the
```

Comment:
To-do -> look up the type of the output of `StatevectorEstimator.run` (it must be some kind of job class)